### DISCUSSION TITLE
What is the web app login credential

### DISCUSSION THREAD

**Discussion by harkamals** [clouditor/clouditor#802] [Q&A] [answered]
When running the command `cl login 127.0.0.1:9090` it generated a url, which I opened in browser is asking for a username, password. What this might be ?

Thanks.

**Comment by oxisto** [comment, marked as answer]
It is `clouditor` / `clouditor` per default. You can also adjust it via the CLI parameters of the `engine` or use another OAuth 2.0 provider.

**Reply by harkamals** [reply]
Thank you :)
